utils/luci-app-ffwizard-berlin/luasrc/tools/freifunk/assistent/tools.lua

```diff
@@ -76,6 +76,22 @@ function firewall_find_zone(name)
 end
 
 
+-- checks if root-password has been set via CGI has_root-pass 
+function hasRootPass()
+	logger ("checking for root-password ...")
+
+	local isPasswordSet = true
+	local f = io.popen("wget http://localhost/cgi-bin/has_root-pass -q -O -")
+	local l = f:read("*a")
+	if l:len()==18 then
```

Comment:
This should check the actual return string, not its length (or add comment explaining what is going on).